projects/libressl/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://www.libressl.org/"
+primary_contact: "libressl-security@openbsd.org"
+auto_ccs:
+ - "bcook@openbsd.org"
```

Comment:
Has this maintainer has agreed to the integration?
(just confirming =)

Reply:
Not yet: @busterb

Reply:
Thanks for the note. Our team is going to be meeting next week, and we can discuss to make sure we're all ready for this integration then. CC @bob-beck @4a6f656c

Reply:
OK. I'll wait to land this until hear back from @busterb

Reply:
Go ahead!

Reply:
After looking for a bit more and discussing with the team, I have some thoughts about where the fuzzer repo lives, and how it is built for testing. I think we want to move the fuzzing base code to a separate repo under the libressl-portable organization and make a few changes in how contacts for results are posted. Give me some more time to get things pulled together. Thanks.

Reply:
Waiting for input! 👍

Reply:
The fuzzing code is now separated out, and fuzzing mailing list setup. @thypon can you please update this PR with pointers to the new resources?